django/contrib/sitemaps/__init__.py

```diff
@@ -44,9 +46,17 @@ def ping_google(sitemap_url=None, ping_url=PING_URL):
     urlopen("%s?%s" % (ping_url, params))
 
 
+def sitemap_time(t):
+    if t is None:
+        return None
+    if isinstance(t, datetime.datetime) and (t.tzinfo is None or t.tzinfo.utcoffset(t) is None):
```

Comment:
Are you sure this isn't duplicating a function in `django.utils.timezone`? Could you please add a docstring to the function?